frontend/templates/layouts/example-pages/amp-stories-page.j2

```diff
@@ -1,4 +1,5 @@
 {% set body_contains_story = example.body and example.has_tag(example.body, 'amp-story') %}
+{% do doc.amp_dependencies.add('amp-story', '0.1') %}
```

Comment:
Sorry, could have mentioned that: all layouts in `frontend/templates/layouts/example-pages` have another dependency mechanism - I guess. How do they get their deps actually, @tharders?

Reply:
@matthiasrohmer @dlemm The imports are directly in these templates:
```html
{% if not example.has_tag_in_head('script', 'custom-element', 'amp-story') %}
  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
{% endif %}
{% for dependency in example.imports if dependency %}
{% if not example.has_import_in_head(dependency) and dependency.name != 'amp-story' %}
  <script custom-element="{{dependency.name}}" src="https://cdn.ampproject.org/v0/{{dependency.name}}-{{dependency.version}}.js" async></script>
{% endif %}
{% endfor %}
```
The example tags have the needed imports as an attribute. The version number is taken from https://playground.amp.dev/api/amp-component-versions

Reply:
Ok, I'll remove the added line.